rebar.config

```diff
@@ -47,8 +47,9 @@
 
 {profiles,
  [{test,
-   [{deps, [{meck, {git, "https://github.com/eproxus/meck.git", {tag, "0.8.2"}}}]}]
+   [{deps, [{meck, {git, "https://github.com/eproxus/meck.git", {tag, "0.8.2"}}}]},
+    {erl_opts, [{d, 'LOG_TO_SYSLOG'}]}]
```

Comment:
This is the cause of the test failures, the `LOG_TO_SYSLOG` controls stdout logging vs syslog logging in a few places, most notably [logplex_logging.hrl](https://github.com/heroku/logplex/blob/d7af36c05294185e4905bc8fbb2cc06a85595f74/src/logplex_logging.hrl#L15)